src/models/index.ts

```diff
@@ -14,3 +14,4 @@ export interface VoteOption {
 }
 
 export type Theme = "light" | "dark";
+export type CookieConsent = "accepted" | undefined
```

Comment:
There was no new line - and github marked it
```suggestion
export type CookieConsent = "accepted" | undefined

```

Reply:
lol